### PR TITLE
WIP Kennric/translate everything

### PR DESCRIPTION
**Pull request by Kennric**
fixes issue #322 

## Changes in this PR.

- [X] Add translation tags for all strings
- [X] Fix settings for correct locale directory location
- [X] Generate message files for Hindi and Spanish
- [X] Make sure tests are correct
- [x] Audit messages to make sure they all make sense.

## Testing this PR.

1. Add translations to one of the supported languages 

### Expected Output.

```
docker-compose run web bash
django-admin compilemessages
exit
docker-compose up web
```

View site and select language you added some translations for. See translated strings.

```
$ make test
[... test output ...]
```

@osuosl/devs

### CODE REVIEW

File: streamwebs_frontend/streamwebs/migrations/0002_auto_20170720_1641.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        ('streamwebs', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='soil_survey',
+            name='site_char',
+            field=models.TextField(blank=True, verbose_name='distinguishing site characteristics'),
+        ),
+    ]
```
Comment (alxngyn) [inline streamwebs_frontend/streamwebs/migrations/0002_auto_20170720_1641.py]: Is this migration file necessary to the PR?

Comment (Kennric) [inline streamwebs_frontend/streamwebs/migrations/0002_auto_20170720_1641.py]: Sadly, changing the verbose name does in fact generate a new migration. It's trivial and petty, but changing the name to remove whitespace make the translation string cleaner.

Comment (alxngyn) [inline streamwebs_frontend/streamwebs/migrations/0002_auto_20170720_1641.py]: oh okay. lgtm

**Comment by JerryPeng0112** [comment]
Have template error for schools.html
++ if it's fixed.

**Pull request merged**